src/pat/tinymce/tinymce--implementation.js

```diff
@@ -232,6 +234,14 @@ export default class TinyMCE {
                     ]);
                 },
             });
+            // handle 'change' event to ensure correct validation (eg. required textfield)
+            // eslint-disable-next-line no-unused-vars
+            editor.on("change", (e) => {
+                // setting tiny content manually
+                this.el.value = editor.getContent();
+                // dispatch "change" event for pat-validation
+                this.el.dispatchEvent(new Event("change"));
```

Comment:
Please use the `change_event` handler from Patternslib because that one bubbles like the original `change` event. Something one often forgets (unless you explicitly don't want to let it bubble).

Do something like:
```
import events from "@patternslib/patternslib/src/core/events";

// ...


                this.el.dispatchEvent(events.change_event());
```